firebase-vscode/src/core/index.ts

```diff
@@ -61,7 +61,7 @@ export async function registerCore(
         workspaceFolder, // The workspace folder
         "firebase init dataconnect", // how you name the task
         "firebase init dataconnect", // Shows up as MyTask: name
-        new vscode.ShellExecution(`${settings.firebasePath} init dataconnect`),
+        new vscode.ShellExecution(`${settings.firebasePath} init dataconnect --project ${currentProjectId.value}`),
```

Comment:
Is this guaranteed to be non-empty when the init button is shown? I thought there's a project selection prompt in the CLI or something